src/utility/Auth.js

```diff
@@ -44,7 +44,7 @@ export function AuthProvider({ children }) {
     analytics.logEvent(ANALYTICS_EVENTS.Logout, { userId: user.uid })
     await firebase.auth().signOut()
     setUser(null)
-    setLocation('/login')
+    setLocation('/')
```

Comment:
why this change?

Reply:
Thought it was a better experience than the current one. On logout you're redirected to `/login`, but it's a "dead end" page since you can't go back in the browser and there's no button for home.